spring-cloud-gcp-autoconfigure/src/main/java/com/google/cloud/spring/autoconfigure/sql/GcpCloudSqlProperties.java

```diff
@@ -40,6 +40,9 @@
 	/** Overrides the GCP OAuth2 credentials specified in the Core module. */
 	private Credentials credentials = new Credentials();
 
+	/** Specifies whether to enable IAM database authentication (PostgreSQL only). */
+	private boolean enableIamAuth;
```

Comment:
maybe just `iamAuth`? `isIamAuth()` reads easier.

Reply:
I'm trying to keep the property name consistent with the socket factory.